palace/utils/configfile.hpp

```diff
@@ -819,6 +819,12 @@ struct LinearSolverData
   // Maximum number of iterations for solving linear systems in divergence-free projector.
   int divfree_max_it = 1000;
 
+  // Relative tolerance for solving linear systems in the error estimator.
+  double estimator_tol = tol;
+
+  // Maximum number of iterations for solving linear systems in the error estimator.
+  int estimator_max_it = max_it;
```

Comment:
Let's give this a default not tied to `max_it`. 100 according to the docs. Also this points out a discrepancy for `div_free_max_it` -> The default in the docs should be 1000, not 100. Can you fix that?